tests/acceptance/commercial/test_delta_update_module.py

```diff
@@ -109,12 +99,51 @@ def test_runtime_checksum(
         reboot(connection)
 
         run_after_connect("true", connection)
-        (active, _) = determine_active_passive_part(bitbake_variables, connection)
-
         connection.run("mender -commit")
 
+        return image
+
+    @pytest.mark.only_with_image("ext4")
+    def test_runtime_checksum(
+        self,
+        request,
+        setup_board,
+        prepared_test_build,
+        bitbake_variables,
+        bitbake_image,
+        connection,
+        http_server,
+        board_type,
+        use_s3,
+        s3_address,
+    ):
+        """Check that the checksum of the running root filesystem is what we
+        expect. This is important in order for it to match when applying a delta
+        update.
+
+        """
+
+        if (
+            "read-only-rootfs"
```

Comment:
Might this as well be a pytest marker?

Reply:
Good idea, but maybe in a later PR :)

This was already present for another test, so I just copy-pasted for now.